hack/k8s-installer.sh

```diff
@@ -63,29 +64,42 @@ check_reachability()
 write_to_extravars()
 {
   sed -i \
-  -e "s/  \"directory\": .*/  \"directory\": \"$directory\",/" \
-  -e "s/  \"build_version\": .*/  \"build_version\": \"$version\",/" \
-  -e "s/  \"release_marker\": .*/  \"release_marker\": \"$release_marker\",/" \
-  -e "s/  \"extra_cert\": .*/  \"extra_cert\" : \"$controllers\",/" \
+  -e "s/^directory: .*/directory: $directory/" \
+  -e "s/build_version: .*/build_version: $version/" \
+  -e "s/release_marker: .*/release_marker: $release_marker/" \
+  -e "s/extra_cert: .*/extra_cert : $(cut -d ' ' -f 1 <<< $controllers)/" \
   $var_file
 }
 
 # Flush contents of hosts.yml file to hold control-plane and worker IPs
 >$hosts_file
 
 # Process arguments
-while getopts "ac:p:rw:y" opt; do
+while getopts "l:ac:p:rw:y" opt; do
   case "$opt" in
     p)
       playbook="$OPTARG";;
     c)
       controllers=("$OPTARG")
       echo "[masters]" >> $hosts_file
-      check_reachability $controllers;;
+      check_reachability "${controllers[@]}";;
     w)
       workers=("$OPTARG")
       echo "[workers]" >> $hosts_file
       check_reachability "${workers[@]}";;
+    l)
```

Comment:
The current design allows only 2+ control-plane nodes to be set up with the load-balancer. There can be a possibility/requirement to set up the load-balancer with a single node too.